docs/content/concepts/logging/loggers.mdx

```diff
@@ -5,37 +5,47 @@ description: Dagster includes a rich and extensible logging system.
 
 # Loggers
 
-Dagster includes a rich and extensible logging system. Dagster comes with a built-in logger that tracks all the execution events. You can also customize loggers to meet your own needs.
+As part of its rich, extensible logging system, Dagster includes loggers. These are job-scoped logging handlers, which will be automatically invoked whenever [ops](/concepts/ops-jobs-graphs/ops) in a [job](/concepts/ops-jobs-graphs/jobs) log messages. This means that this out-of-the-box logger will track all execution events. Loggers can also be customized to meet your specific needs.
```

Comment:
So the loggers passed to `Definitions` are applied to *all* jobs within it, and they can be overridden at the job-level. I think we should frame job-scoped logger objects as an advanced feature.